wsnet/listen.go

```diff
@@ -316,3 +336,58 @@ func (l *listener) Close() error {
 func (l *listener) Addr() net.Addr {
 	return nil
 }
+
+// normalizeHost converts all representations of "localhost" to "localhost".
+func normalizeHost(addr string) string {
```

Comment:
Calling `net.LookupIP` might be a cleaner way to standardize the URLs.

Reply:
I don't want to make DNS requests from this. If people want to do `localhost.example.com` then their connections will just get aborted. They can just use `localhost` or `127.0.0.1` instead.

Reply:
Fair enough fair enough.